docs/source/contributing/testing.rst

```diff
@@ -1,21 +1,24 @@
 ============
 Adding Tests
 ============
-When adding a new feature, it should always be tested. Tests prevent
-manim from breaking at each new feature added by checking if any other
+If you are adding new features to manim, you should add appropriate tests for them. Tests prevent
+manim from breaking at each change by checking that no other
 feature has been broken and/or been unintentionally modified.
 
 How Manim Tests
 ---------------
 
-To conduct our tests, we use ``pytest``. Running ``pytest`` in the root of
-the project will start the testing process, and will show if there is
-something wrong.
+Manim uses pytest as its testing framework. 
+To start the testing process, go to the root directory of the project and run pytest in your terminal. 
+Any errors that occur during testing will be displayed in the terminal.
 
 Some useful pytest flags: 
-- ``-x``, that will make pytest stop at the first fail,
-- ``-s``, that will make pytest display all the print messages (including those during scene generation, like DEBUG messages).
-- ``--skip_slow`` will skip the (arbitrarly) slow tests. 
+- ``-x`` will make pytest stop at the first failure it encounters
+  
+- ``-s`` will make pytest display all the print messages (including those during scene generation, like DEBUG messages)
+  
+- ``--skip_slow`` will skip the (arbitrarily) slow tests
+  
 - ``--show_diff`` will show a visual comparison in case an unit test is
 failing. 
```

Comment:
This list is unfortunately not correctly formatted (see https://manimce--1217.org.readthedocs.build/en/1217/contributing/testing.html):

- there should be a newline before the first entry
- "failing" has to be indented in order to be rendered as part of the list item. Otherwise the line break could be removed.

Reply:
Should be fixed now.